### PR TITLE
Don't embed JARs installed in lib/jars

### PR DESCRIPTION
**Pull request by bjorncs**
Define installed JARs in vespa-3party-jars. Add bundle-plugin goal wrapping maven-shade-plugin's DefaultShader that excludes installed JARs and lists them in manifest's Class-Path instead.

### CODE REVIEW

**Review by baldersheim — APPROVED** [review]
🥇

**Review by gjoranv — APPROVED** [review]
Great! Just one question.

File: bundle-plugin/src/main/java/com/yahoo/container/plugin/mojo/AssembleFatJarMojo.java [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+            "META-INF/MANIFEST.MF",
+            "META-INF/*.SF",
+            "META-INF/*.DSA",
+            "META-INF/*.RSA",
+            "about.html",
+            "module-info.class",
+            "license/*",
+            "**/package-info.class",
+            "**/module-info.class",
+    };
```
[the remark this answers is not in the document]
Comment (bjorncs) [reply bundle-plugin/src/main/java/com/yahoo/container/plugin/mojo/AssembleFatJarMojo.java:84]: It's overridable, but the defaults must then be reapplied manually. It's a possible to extend the mojo with same filtering API as shade plugin (can reuse some filter configuration classes).

[the remark this answers is not in the document]
Comment (gjoranv) [reply bundle-plugin/src/main/java/com/yahoo/container/plugin/mojo/AssembleFatJarMojo.java:84]: Couldn't this list be separate from the parameter, if the main case is to add?

**Pull request merged**